donate/utility/acoustic/acoustic.py

```diff
@@ -0,0 +1,126 @@
+import logging
```

Comment:
Code to send emails through acoustic, in AcousticTransact's "send_mail" function, we are checking whether or not the "IS_TESTING" testing env variable is set to true, that way we dont send emails when testing

Reply:
I'm not certain we need the IS_TESTING env variable. I believe the correct way would be to mock this function in our test cases. @Pomax might be able to help you with that. Let's push this for now, but we'll need to make a ticket to find a better solution when Pomax is back.

Reply:
This was my suggestion based on the deadline, with the same comment to look into this in the future.